build.gradle.kts

```diff
@@ -207,6 +207,13 @@ allprojects {
     }
     tasks.withType<Test>().configureEach {
         useJUnitPlatform()
+        this.testLogging {
+            events("failed")
+            exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+            showExceptions = true
+            showCauses = true
+            showStackTraces = true
+        }
```

Comment:
Added this bc it's much easier to see failing tests in the terminal than having to navigate out to a browser link to see the failing tests.